tests/unit/test_models.py

```diff
@@ -60,3 +60,14 @@ def test_duplicated_whiteboard_tag_fails(action_factory):
             ]
         )
     assert "actions have duplicated lookup tags: ['x']" in str(exc_info.value)
+
+
+def test_step_overriding():
```

Comment:
maybe add a comment or rename to `test_single_step_list_overriding` or something like that?

Reply:
[aad5647](https://github.com/mozilla/jira-bugzilla-integration/pull/591/commits/aad56475ee99c95edfa84f828dce994b71fb4752)